### PR TITLE
Add timeToMergeFromFirstReviewSeconds

### PR DESCRIPTION
**Pull request by odanado**
Thank you for a great cli.

timeToMergeFromFirstReviewSeconds: seconds between a first review  and a PullRequest merged.
timeToMergeFromFirstReviewSeconds represents the time spent reviewing the pull request.

This allows for a more detailed analysis of the lead time breakdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ output is
 * leadTimeSecondsMedian: the median of seconds between a first commit date and a PullRequest merged date
 * timeToMergeSecondsAverage: the average of seconds between a PullRequest created and a PullRequest merged
 * timeToMergeSecondsMedian: the median of seconds between a PullRequest created and a PullRequest merged
+* timeToMergeFromFirstReviewSecondsAverage: the average of seconds between a first review  and a PullRequest merged.
+* timeToMergeFromFirstReviewSecondsMedian: the median of seconds between a first review  and a PullRequest merged.
```
Comment (shibayu36) [inline README.md:46]: Please add these fields to json sample above

**Review by shibayu36 — CHANGES_REQUESTED** [review]
It's good fields to understand productivity.  I suggested to fix README.md

**Comment by odanado** [comment]
@shibayu36 I have fixed the README. Could you re-review.

**Review by shibayu36 — APPROVED** [review]
Great! LGTM

**Comment by shibayu36** [comment]
I have just released it as 0.3.0
https://www.npmjs.com/package/@shibayu36/merged-pr-stat

Thanks for your pull request

**Pull request merged**